module/documents/actor.mjs

```diff
@@ -374,71 +362,65 @@ export default class ActorEd extends Actor {
     let strain = 0;
     let knockdownStep = attributes.str.step;
 
-    const knockdownAbilities = this.items.filter( item => item.system.edid === "knock-down" );
-
-    if ( knockdownAbilities.length > 0 ) {
-      const knockdownAbilityId = await KnockDownItemsPrompt.waitPrompt( knockdownAbilities );
+    const knockdownAbility = await fromUuid(
```

Comment:
@ChristopherSD this returns always null even though i have three items with knock-down ed id. The buttons are there, but the abilities are not used.

Reply:
wierd thing... the step is one higher than the attribute step even though I don't have an ability